### PR TITLE
limit data access with domain keys, remove generation

### PR DESCRIPTION
**Pull request by trieloff**
This PR adds support for domain keys parameters, and removes the generation filter from most queries

### CODE REVIEW

**Comment by trieloff** [comment]
@tripodsan the SQLFluff errors are due to parsing issues for some parts of BigQuery SQL that SQLFluff does not know about yet.

File: src/index.js [hunk 1]
```diff
@@ -71,9 +70,8 @@ async function runExec(params, pathname, log) {
 async function run(request, context) {
   const { pathname } = new URL(request.url);
   const params = context.data;
-  params.token = request.headers.has('x-token') ? request.headers.get('x-token') : undefined;
   /* c8 ignore next */
-  params.service = request.headers.has('x-service') ? request.headers.get('x-service') : undefined;
+  params.domainkey = request.headers.has('authorization') ? request.headers.get('authorization').split(' ').pop() : params.domainkey || 'secret';
```
Comment (tripodsan) [inline src/index.js:74]: this is a bit weak.... why use the split if you don't need/check the first part?

**Review by tripodsan — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (trieloff) [reply src/index.js:74]: Would you use a regex?

**Pull request merged**